### ISSUE TITLE
where is the loss fuction

### ISSUE THREAD

**Issue by xwr96** [JJN123/Fall-Detection#9]
hi! I can not found the loss fuction in your code,I only see this `autoencoder.compile(optimizer='adadelta', loss='mean_squared_error')`. So if I want to change the loss fuction,what should I do? Or where the loss fuction is?
thanks!

**Comment by shehrozskhan** [comment]
As the code shows, the loss is "mean squared error", which is the mean of squared difference between input and output.
If you want to write your own loss function, please follow KERAS documentation on "Creating custom losses" here --> https://keras.io/api/losses/.

**Comment by xwr96** [comment]
but as the paper said,the loss formula is as follow:
![Screenshot from 2021-06-15 21-56-32](https://user-images.githubusercontent.com/28647688/122065688-b8d45600-ce24-11eb-8740-6283748ca6bd.png)
I want to know where  the code is？

**Comment by shehrozskhan** [comment]
This is not the formula for loss, rather it is to calculate the anomaly score. You will want to look into util.py and the following functions below.
def make_cross_window_matrix(scores):
def agg_window(RE, agg_type):
def gather_auc_avg_per_tol(inwin_mean, inwin_std, labels, win_len = 8):
